guide/inheritance-wallet/making-changes.md

```diff
@@ -0,0 +1,283 @@
+   image = "/assets/images/guide/inheritance-wallet/making-changes/key-replacement/prototype-key-replacement.jpg"
+   retina = "/assets/images/guide/inheritance-wallet/making-changes/key-replacement/prototype-key-replacement@2x.jpg"
+   mobile = "/assets/images/guide/inheritance-wallet/making-changes/key-replacement/prototype-key-replacement-mobile.jpg"
+   mobileRetina = "/assets/images/guide/inheritance-wallet/making-changes/key-replacement/prototype-key-replacement-mobile@2x.jpg"
+   alt-text = "An image showing couple of reference screens from the prototype"
+   width = 800
+   height = 500
+%}
+
+On the wallet settings page, Bob goes the signing keys page. From there the app guides him through the key replacement flow. It starts with a short explanation dialog that outlines how the process works.   
+
+{% include image-gallery.html pages = page.images_key-replacement %} 
+
+The app then asks which of the keys Bob would like to replace. Bob selects David's Passport from the list of keys. On the next screen, the app gives Bob the option to change the wallet configuration as well. But Bob only needs to replace David's key, so he continues. He does, however, select a different name for the new wallet.
+
+The next step is to add David's new extended public key. This works in the same way as during the initial wallet creation. On the wallet review screen, the app indicates that a new signing key has been added. Bob reviews the changes and hits "create wallet."
+
+### Cosigner setup, device activation and wallet backup 
+The new inheritance wallet is now created. Just like during the initial wallet creation, Alice and Bob now need to:
+
+1. Share the wallet configuration with Alice, so that she can set up the wallet on her app.
+2. Activate the new wallet on their signing devices.
+3. Save the backup kit for the new wallet.
+
+We have covered these topics in the previous pages of this reference design.
+
+### Moving funds to the new wallet
+
+Now that the new wallet is ready, Alice and Bob need to transfer their savings to this new wallet. Again, our app shows this in the form of a reminder. 
+
+#### Bob creates and signs the transaction
+
+Bob taps on the reminder and starts creating the transaction. Because the app knows about both wallets, it will prefill most of the information in the send flow, like the recipient address and the transaction amount. 
+
+{% include image-gallery.html pages = page.images_fund-transfer-bob %} 
+
+After creating the transaction, Bob directly continues to sign it with his Trezor. Once this is done, Alice will provide her signature.
+
+#### Alice co-signs the transaction
+
+On the app home screen, Alice now sees a reminder that a transaction is ready to be signed. On the detail screen, she also sees that Bob has already provided a signature. Alice hits "sign" and goes through the transaction signing flow on her Bitbox. 
```

Comment:
> now sees a reminder that a transaction is ready to be signed

How does the app know about the transaction? Is there a server handling communication? If not, then Bob would have to share the PSBT.